components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Registry/src/Deploy/IoTHubDiscovererDeployment.cs

```diff
@@ -115,20 +115,19 @@ public sealed class IoTHubDiscovererDeployment : IHostProcess {
                 });
             }
             else {
-                // Windows
+                // Windows (Eflow)
```

Comment:
Does this mean we discontinue windows containers with this version? if so, we should probably add a little migration tutorial.

Reply:
On a second thought, if we discontinue windows containers, then the Linux vs Windows configurations should be identical, therefore the entire logic here could be optimized.